lib/tasks/summaries.rake

```diff
@@ -0,0 +1,43 @@
+require "happy_hood/slack/client"
+
+namespace :summaries do
+  desc "Send daily summary to Slack"
+  task daily: :environment do
```

Comment:
So the idea is we'll schedule this to run some time after the `house_valuation_collector` task runs?

Reply:
Yeah! The `house_valuation_collector` can run anywhere after midnight, so I figured sometime around 6 AM CST.